### PR TITLE
WTF is up with these log APIs?

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

**Comment by jerith** [comment]
They're broken, that's what's up with them. 😡

File: seaworthy/tests-core/test_logs.py [hunk 1]
```diff
@@ -5,10 +5,23 @@
 from datetime import datetime
 
 from seaworthy._lowlevel import stream_logs
+from seaworthy.checks import docker_client, dockertest
+from seaworthy.dockerhelper import DockerHelper, fetch_images
 from seaworthy.logs import (
     EqualsMatcher, RegexMatcher, SequentialLinesMatcher,
     stream_with_history, wait_for_logs_matching)
 
+# We use this image to test with because it is a small (~7MB) image from
```
Comment (JayH5) [inline seaworthy/tests-core/test_logs.py]: ~4MB?

File: seaworthy/tests-core/test_logs.py [hunk 2]
```diff
@@ -421,3 +434,108 @@ def test_kwargs(self):
...
+@dockertest()
+class TestWithRealContainer(unittest.TestCase, FakeAndRealContainerMixin):
+    def setUp(self):
+        self.dh = DockerHelper()
+        self.addCleanup(self.dh.teardown)
+        self.dh.setup()
+
+    def start_logging_container(self):
+        script = '\n'.join([
+            'for n in 0 1 2 3 4 5 6 7 8 9; do',
```
Comment (JayH5) [inline seaworthy/tests-core/test_logs.py]: `for n in $(seq 0 9)`? (Not that it really matters)

Comment (jerith) [reply seaworthy/tests-core/test_logs.py]: I don't want to rely on `seq` being available -- it's not POSIX (BSD-derived systems generally have `jot` instead) and thus may not be present in arbitrary containers.

File: seaworthy/tests-core/test_logs.py (hunk 1 above)
Comment (jerith) [reply seaworthy/tests-core/test_logs.py]: Thanks.

**Pull request merged**